Automattic/wp-calypso
# Vertical scrolled plans for mobile A/B test

#### Changes proposed in this Pull Request

* Based on project thread pcbrnV-XN-p2
* Provides a vertically scrollable card view for mobile resolutions in the signup flow
* Add a "device" prop to events related funnel to be analysed which can have values ["DESKTOP", "MOBILE", "TABLET"]

| Mobile            | IPAD               |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/3422709/104597435-558aa000-569b-11eb-95d4-289141402932.png)  | ![image](https://user-images.githubusercontent.com/3422709/104597533-7521c880-569b-11eb-8ef8-49f21ec31b56.png)  |


### Testing instructions

- ~~Hardcode experiment to be activated set true for `isInVerticalScrollingPlansExperiment`~~
~~ https://github.com/Automattic/wp-calypso/pull/48728/files#diff-f4523a33abda8414624dfd8061cffa9cc3b52e263f9a755706831b28b7b55cecR324-R325 ~~

- Activate the experiment that has been setup on ExPlat. Check pcbrnV-XN-p2 for link.
- Login to wordpress 
- In one tab Goto  /start and go to the plans step (Sign up flow plans step)
- In another go to the home page and go to the plan page
<img src="https://user-images.githubusercontent.com/3422709/104598724-eada6400-569c-11eb-8317-73c5f7b7a460.png" width="400" >

- Switch to mobile resolutions in both tabs and check whether the styles match across both mobile views (Width < 1045px)
- Make sure the sign up flow plans page in the desktop resolution remains unchanged in caparison with prod
- Make sure the logged in plans page remains unchanged in caparison with prod

#### Test to make sure control remains unchanged
- Activate control in experiment
- Make sure the signup flow plans page as well as the logged in plan page match production exactly
